vladyslav.taran/Home_work1/task_1/1.rb

```diff
@@ -0,0 +1,3 @@
+p arr = Array(3) {Array(3) {rand (-10..10)}}
```

Comment:
This line of code generates an array with 1 random number instead of 3 arrays with 3 random numbers.

```ruby
# actuale result
Array(3) {Array(3) {rand (-10..10)}}
=>[3]

# expected result
=> [
  [-1, 2, 4],
  [3, 4, -5],
  [-9, 9, 1]
]
```

Reply:
Check how work `Array.new(3) { some code instide block }`. The main problem with this line that you don't call the method `new` on class `Array`